tools/integration_tests/read_cache/remount_test.go

```diff
@@ -64,18 +67,18 @@ func readFileAndValidateCacheWithGCSForDynamicMount(bucketName string, ctx conte
 // Test scenarios
 ////////////////////////////////////////////////////////////////////////
 
-func (s *remountTest) TestCacheClearsOnRemount(t *testing.T) {
+func (s *remountTest) TestCacheIsNotReusedOnRemount(t *testing.T) {
 	testFileName := setupFileInTestDir(s.ctx, s.storageClient, testDirName, fileSize, t)
 
 	// Run read operations on GCSFuse mount.
-	expectedOutcome1 := readFileAndValidateCacheWithGCS(s.ctx, s.storageClient, testFileName, fileSize, t)
-	expectedOutcome2 := readFileAndValidateCacheWithGCS(s.ctx, s.storageClient, testFileName, fileSize, t)
+	expectedOutcome1 := readFileAndValidateCacheWithGCS(s.ctx, s.storageClient, testFileName, fileSize, false, t)
```

Comment:
We could have kept checkCacheSize true here (line 74 and 75)

Reply:
+1

Reply:
Done